### DISCUSSION TITLE
Is it possible to specify a destination folder?

### DISCUSSION THREAD

**Discussion by raelb** [gildas-lormeau/SingleFile#741] [Q&A]
**Is your feature request related to a problem? Please describe.**
Saving to the default download folder on Windows (`C:\Users\User\Downloads\`), means that SingleFile downloads are mixed up with other file downloads.

**Describe the solution you'd like**
I would prefer setting a specific folder to save SingleFile downloads to. It would make managing these files easier

I'm on Windows 10 x64.
Thanks

**Comment by gildas-lormeau** [comment]
You can save the pages in a sub-folder of the default download folder by prepending its name to the `File name > template` option, e.g. `SingleFile/{page-title} ({date-locale} {time-locale}).html` to save pages into `C:\Users\User\Downloads\SingleFile`.

**Reply by GCRaistlin** [reply]
How do I change the default download folder to a folder that is not a subfolder of the default profile download folder?

**Reply by gildas-lormeau** [reply]
You have to use an additional external program for this, see https://github.com/gildas-lormeau/single-file-companion-lite.
